extensions/libraries/redcore/api/api.php

```diff
@@ -261,10 +261,12 @@ public static function getPostedData()
 			$inputData = $input->post->getArray();
 		}
 
-		// Filter data with JInput default filter
-		$postedData = new JInput($inputData);
+		$filter = JFilterInput::getInstance(array(), array(), 1, 1, 1, -1);
```

Comment:
I don't know maybe we are looking at a different class. The one I'm referring to is https://github.com/joomla/joomla-cms/blob/staging/libraries/joomla/filter/input.php#L188